pulsar/client.go

```diff
@@ -142,6 +142,8 @@ type ClientOptions struct {
 	// Specify metric registerer used to register metrics.
 	// Default prometheus.DefaultRegisterer
 	MetricsRegisterer prometheus.Registerer
+
+	IsEnableTransaction bool
```

Comment:
Could you remove the `Is` prefix? See other options that enable some feature:
- producer.go: EnableChunking
- consumer.go: EnableBatchIndexAcknowledgment, RetryEnable, EnableDefaultNackBackoffPolicy